### DISCUSSION TITLE
Impact of CVE-2023-32321 in CKAN 2.7.2

### DISCUSSION THREAD

**Discussion by Gauravp-NEC** [ckan/ckan#7642] [General]
@amercader @YoloClin,  With reference to CVE-2023-32321(https://github.com/ckan/ckan/security/advisories/GHSA-446m-hmmm-hm8m)
The affected versions are <2.9.9, <2.10.1 are mentioned in CVE, So v2.7.2 is also affected by this CVE?

**Comment by amercader** [comment]
The CVE advisory clearly lists the affected and patched versions. Any version lower than the ones listed as patched is affected by the vulnerability.

CKAN 2.7 is not supported and hasn't been for a very long time so I strongly encourage you to upgrade to a supported version as soon as possible
